draft-ietf-sedate-datetime-extended.md

```diff
@@ -196,16 +196,62 @@ Internet Date/Time Format:
 Timestamp:
 : An unambiguous representation of some instant in time.
 
+UTC Offset:
+: Difference between a given local time and UTC, usually given in
+  negative or positive hours and minutes. For example, local time in New
+  York in the wintertime is 5 hours behind UTC, so its UTC offset is "-05:00".
+
 Z:
 : A suffix which, when applied to a time, denotes a UTC offset of
-  00:00; often spoken "Zulu" from the ICAO phonetic alphabet
+  +00:00; often spoken "Zulu" from the ICAO phonetic alphabet
```

Comment:
This is the "+" that appears to be an unnecessary addition to the definition as cited from RFC 3339.

Reply:
Sounds good, I'd assumed it was a typo, but if there's a good reason to leave as-is that sounds fine to me.

Reply:
I too prefer to be explicit but it's nice to keep the diff smaller.

Reply:
Note that every offset other than 00:00 does need a sign, and which sign you use with 00:00 makes no difference, so it is a kindness to the authors of parsers for the spec to always insist there be a sign, saving them the need to special-case the no-sign 00:00.

Reply:
A kindness not afforded by the text of RFC 3339.  This is a definition of the term "Z", copied verbatim from RFC 3339.  Changing or not changing this definition in this draft will not change the syntax for timestamps, which already requires + or - to delimit the offset from the time.